applications/GeoMechanicsApplication/custom_elements/README.md

```diff
@@ -0,0 +1,105 @@
+
+$$ \left(n S \rho^w c^w + (1- n) \rho^s c^s \right) \frac{\partial T}{\partial t} = -\rho^w c^w q_i \frac{\partial T}{\partial x_i} + \frac{\partial}{\partial x_i} \left( D_{ij} \frac{\partial T}{\partial x_j} \right) \qquad \text{on} \quad \Omega $$
+
+where,
+
+- $c^w$       = specific heat capacity liquid phase $\mathrm{[J/kg ^{\circ}C]}$
+- $c^s$        = specific heat capacity solid phase  $\mathrm{[J/kg ^{\circ}C]}$
+- $D_{ij}$     = hydrodynamic thermal dispersion  $\mathrm{[W/m ^{\circ}C]}$
+- $T$           = temperature  $\mathrm{[ ^{\circ}C]}$
+- $\rho^s$ = density solid phase $\mathrm{[kg/m^3]}$ 
+
+The hydrodynamic thermal dispersion is defined as:
+
+$$ D_{ij}= nS \lambda^w \delta_{ij} + \left(1-n\right) \lambda_{ij}^s + c^w \rho^w \left( (\alpha_l - \alpha_t) \frac{q_i q_j}{q} + \delta_{ij} \alpha_t q \right) $$
+
+where
+
+- $\alpha_l$    = longitudinal dispersivity $\mathrm{[m]}$
+- $\alpha_t$    = transverse dispersivity $\mathrm{[m]}$
+- $\delta_{ij}$ = Kronecker delta $\mathrm{[-]}$
+- $\lambda^w$   = thermal conductivity water $\mathrm{[W/m ^{\circ}C]}$
+- $\lambda^s$   = thermal conductivity solid matrix $\mathrm{[W/m ^{\circ}C]}$
+
+In the absence of ground water flow, these equations are simplified to,
+
+$$ \left(n S \rho^w c^w + (1- n) \rho^s c^s \right) \frac{\partial T}{\partial t} = \frac{\partial}{\partial x_i} \left( D_{ij} \frac{\partial T}{\partial x_j} \right) \qquad \text{on} \quad \Omega $$
+
+$$ D_{ij}= nS \lambda^w \delta_{ij} + \left(1-n\right) \lambda_{ij}^s $$
+
+## Boundary Conditions
+
+### Dirichlet boundary condition
+$$ T = \overline T \qquad \text{on} \quad \Gamma_{1}^T $$
+
+where $\overline T$ $\mathrm{\left[ ^{\circ}C \right]}$ is a prescribed temperature 
+
+### Neumann Boundary Condition
+$$ D_{ij} \frac{\partial T}{\partial x_j} n_i = \overline{f} \qquad \text{on} \quad \Gamma_{2}^T $$
+
+where $\overline f$ $\mathrm{\left[ W/m^2 \right]}$ is a prescribed conductive heat flux.
+
+### Robin bounday condition
+$$ D_{ij} \frac{\partial T}{\partial x_j} n_i = \overline{g} -  \rho^w c^w q_n T \qquad \text{on} \quad \Gamma_{3}^T $$
+
+where $\overline g$ $\mathrm{\left[ W/m^2 \right]}$ is a prescribed convective-conductive heat flux.
+
+
+## Derived Properties
+
+The density of the bulk material $\rho$ $\mathrm{\left[ kg/m^3 \right]}$ is calculated as,
+
+$$ \rho = n S \rho^w + \left( 1 - n \right) \rho^s $$
+
+And the heat capacity of the bulk material $C$ $\mathrm{\left[ J/m{^3 \circ}C \right]}$ is:
+
+$$ C = n S \rho^w c^w + \left( 1 - n \right) \rho^s c^s $$
+
+The thermal conductivity of the bulk material $\lambda$ $\mathrm{\left[ W/m ^{\circ}C \right]}$
+
+$$ \lambda = n S \lambda^w + \left( 1 - n \right) \lambda^s $$
+
+dynamic viscosity  $\mu$ $[\mathrm {Pas}$] of pure water as a function of temperature is given by:
```

Comment:
Are the 2 following temperature dependencies always used or can a user choose?
This always makes the computation depend on this choice of units.